.vscode/launch.json

```diff
@@ -20,4 +20,4 @@
             "runtimeExecutable": "ts-node",
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
nit, unnecessary change? I've noticed prettier will try to remove/add whitespace in json files. you can use the vs code command center to save without formatting to prevent these changes from happening.